code/def_files/data/effects/main-f.sdr

```diff
@@ -293,10 +293,17 @@ void main()
  #ifdef FLAG_TEAMCOLOR
 	vec4 teamMask = vec4(0.0, 0.0, 0.0, 0.0);
 	teamMask = texture(sMiscmap, vec3(texCoord, float(sMiscmapIndex)));
-	vec3 base = max(base_color - vec3(0.5), vec3(0.0));
-	vec3 stripe = max(stripe_color - vec3(0.5), vec3(0.0));
-	baseColor.rgb += (base * teamMask.x) + (stripe * teamMask.y);
+	vec3 team_color = ((base_color - vec3(0.5)) * teamMask.x) + ((stripe_color - vec3(0.5)) * teamMask.y);
```

Comment:
I think it would be better to keep using the original `base` and `stripe` variables to reduce the amount of recompuatation of the adjusted color values. Also the new version is missing the `max` function call from the original version.

Reply:
That is intentional. That max() call was responsible for making team colors not show up as strongly as they are intended to; I am not sure why this looked correct on the old renderer but doesn't here.